documentation for msys.txt

```diff
@@ -0,0 +1,8 @@
+Msys2:

```

Comment:
This is instructions how to built raqm with Msys2. I think it should move to readme.txt